### PR TITLE
Add more documentation

### PR DESCRIPTION
**Pull request by ollytheninja**

### CODE REVIEW

**Comment by rupertbg** [comment]
I think writing out a file in json or yaml with the stats is a great idea

File: generator/stats.go [hunk 1]
```diff
@@ -50,9 +47,6 @@ Most common action prefixes:
 {{- range .Prefixes }}
 | {{ $.Backtick }}{{ .Prefix }}{{ $.Backtick }} | {{ .Count }} |
 {{- end }}
-
-Thank you to [alanakirby/aktion](https://github.com/alanakirby/aktion) for originally
```
Comment (rupertbg) [inline generator/stats.go]: oi! y u no

Comment (ollytheninja) [reply generator/stats.go]: Because it's in README.md not stats.md

Comment (ollytheninja) [reply generator/stats.go]: https://github.com/glassechidna/trackiam/pull/17/files#diff-04c6e90faac2675aa89e2176d2eec7d8R13

**Pull request merged**